src/libraries/Common/src/System/Drawing/ColorConverterCommon.cs

```diff
@@ -37,7 +37,7 @@ public static Color ConvertFromString(string strValue, CultureInfo culture)
             // If the value is a 6 digit hex number only, then
             // we want to treat the Alpha as 255, not 0
             //
-            if (text.IndexOf(sep) == -1)
+            if (!text.Contains(sep, StringComparison.Ordinal))
```

Comment:
The correct replacement of `IndexOf(char)` should be `Contains(char)`. `Contains(char, StringComparison.Ordinal)` will just call `IndexOf(char)` which is what you want replaced.